src/pages/publication/PublicationForm.tsx

```diff
@@ -25,13 +28,15 @@ const StyledPublication = styled.div`
 
 interface PublicationFormProps {
   uppy: Uppy;
+  closeForm: () => void;
   id?: string;
 }
 
-const PublicationForm: FC<PublicationFormProps> = ({ uppy = createUppy(shouldAllowMultipleFiles), id }) => {
+const PublicationForm: FC<PublicationFormProps> = ({ uppy = createUppy(shouldAllowMultipleFiles), id, closeForm }) => {
```

Comment:
Bør kanskje bruke identifier her istedet for id hvis man endrer det andre steder også.

Reply:
good spot!